src/data/events.json

```diff
@@ -3,9 +3,10 @@
     "eventName": "Kubernetes Community Days Chennai 2025",
     "eventDescription": "Kubernetes Community Days Chennai 2025 is a community-organized conference supported by the Cloud Native Computing Foundation (CNCF) to help grow and sustain the Kubernetes and cloud-native community in Chennai. It gathers adopters and technologists from open source and cloud-native communities for education, collaboration, and networking.",
     "eventDate": "2025-04-26",
-    "eventTime": "08:30",
-    "eventVenue": "Sathyabama University",
-    "eventLink": "https://community.cncf.io/kcd-chennai/",
+    "eventStartTime": "08:30",
```

Comment:
just like I mentioned above, lets not complicate things with more information, we can display just the start time and that should be good